Packs/ImpossibleTraveler/ReleaseNotes/1_2_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Impossible Traveler
+- Updated the Docker image to: *demisto/py3-tools:0.0.1.25751*.
```

Comment:
the release note should state the script/integration that was updated, as the playbook itself was not updated.
see ` FetchIndicatorsFromFile/ReleaseNotes/1_0_5.md` in your PR as a good example.